pshtt/pshtt.py

```diff
@@ -521,9 +559,24 @@ def hsts_check(endpoint):
             return
```

Comment:
It's not just a bad hostname that should cause the HSTS header to be ignored.  It is also ignored if the certificate is untrusted, for example.  (See item 2 in [the RFC](https://tools.ietf.org/html/rfc6797#section-2.2).)  Should this `if` clause be changed?

Reply:
@jsf9k Shouldn't that be in a separate issue/pull request to only give credit for HSTS if the endpoint's certificate is trusted?  Currently we don't check that the certificate is trusted even on the initial response, so that change could have more negative effects on the results.  This change here to check more responses from the same endpoint can only have positive results by giving some sites credit for HSTS that don't get credit for it today, so I think we should finish this one first and then make that change separately.

Reply:
@jsf9k Based on @h-m-f-t 's comment at https://github.com/cisagov/pshtt/issues/89#issuecomment-318242757 saying 

> I don't want compliance for compliance' sake, or maximal-compliance for a little security benefit— and the current approach does have a bit of that flavor. Happy to hear more discussion, but I think I'm supportive of making this change, especially given that SSL Labs does this.

I think @h-m-f-t is already supportive of making this change.

Reply:
> Shouldn't that be in a separate issue/pull request to only give credit for HSTS if the endpoint's certificate is trusted?

You're right.  I'll create a separate issue for that.

Reply:
> @jsf9k Based on @h-m-f-t 's comment at [#89 (comment)](https://github.com/cisagov/pshtt/issues/89#issuecomment-318242757) saying
> 
> > I don't want compliance for compliance' sake, or maximal-compliance for a little security benefit— and the current approach does have a bit of that flavor. Happy to hear more discussion, but I think I'm supportive of making this change, especially given that SSL Labs does this.
> 
> I think @h-m-f-t is already supportive of making this change.

Understood, but I'd like him to be more explicit about it by approving this pull request.  In any event, we need two non-author approvals to merge it.